src/mokka/src/main/java/pl/hycom/mokka/endpoint/FileController.java

```diff
@@ -55,7 +58,7 @@
         LOG.debug("Calling FileController#fetchFile with arguments [" + fileId + "]");
         File file = fileService.fetchFile(fileId);
         HttpHeaders headers = new HttpHeaders();
-        headers.add("Content-Disposition", "attachment; filename=" + file.getName());
+        headers.add("Content-Disposition", contentDisposition+"; filename=" + file.getName());
```

Comment:
Good idea with parametrization.
After some thinking, I would say we can go even further. As we allow to get any type of file, one type of Content-Disposition (inline or attachment) will not work (i.e. inline for zips). So maybe it would be good idea to leave this decision for API consumer? 
My proposition is to add optional request parameter for this. If not provided value from `files.contentDisposition` will be used.